.editorconfig

```diff
@@ -0,0 +1,19 @@
+root = true
```

Comment:
I love the fact you add this

But I would expect to see a GitHub action for validating the files added via a future PR will be checked

I would like to recommend

https://github.com/editorconfig-checker/action-editorconfig-checker